internal/commands/accesslist/update_test.go

```diff
@@ -0,0 +1,207 @@
+package accesslist
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+)
+
+func TestAllowedIPUpdateHandler(t *testing.T) {
+	projectID := "projectID"
+	appID := "appID"
+	app := realm.App{
+		ID:          appID,
+		GroupID:     projectID,
+		ClientAppID: "test-abcde",
+		Name:        "test",
+	}
+
+	allowedIPs := []realm.AllowedIP{
+		{ID: "address1", Address: "0.0.0.0", Comment: "comment"},
+		{ID: "address2", Address: "192.1.1.1"},
+		{ID: "address3", Address: "192.158.1.38", Comment: "cool comment"},
+	}
+
+	for _, tc := range []struct {
+		description    string
+		testAddress    string
+		testNewAddress string
+		testComment    string
+	}{
+		{
+			description:    "should return a successful message for a successful update for address and comment",
+			testAddress:    "0.0.0.0",
+			testNewAddress: "2.2.2.2",
+			testComment:    "new comment",
+		},
+		{
+			description:    "should return a successful message for an update with only an address",
+			testAddress:    "192.1.1.1",
+			testNewAddress: "68.192.33.2",
+		},
+		{
+			description: "should return a successful message for an empty update",
```

Comment:
should we be testing this scenario if its not technically possible based on the way inputs are resolved?

Reply:
noticed that secrets was testing it like this too, but yeah I agree we don't necessarily need to be testing this functionality since it isn't a possible scenario.

Reply:
could probably drive-by remove that one as well then